src/git.md

```diff
@@ -347,9 +346,9 @@ you might want to get used to the main concepts of Git before reading this secti
 
 The `rust-lang/rust` repository uses [Git submodules] as a way to use other
 Rust projects from within the `rust` repo. Examples include Rust's fork of
-`llvm-project` and many devtools such as `cargo`, `rust-analyzer` and `rustfmt`.
+`llvm-project` and many devtools such as `cargo`, `rust-analyzer` and `RLS`.
```

Comment:
```suggestion
`llvm-project` and many devtools such as `cargo`, `rust-analyzer` and `rls`.
```
I assume? It's lower-case most everywhere else.

Reply:
done!